<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASL232</class>
 <widget class="QWidget" name="ASL232">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1,1,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>通道号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxChannelSend"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxBaudrate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxDataBit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxStopBit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>校验位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxParityBit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonConfig">
           <property name="text">
            <string>配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelSendLength">
           <property name="text">
            <string>数量</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEditSend"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClear">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTimerSet"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxFixData">
             <property name="text">
              <string>固定数据</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxTimerSet">
             <property name="text">
              <string>定时发送</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSend">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>通道号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxChannelRecv"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>接收</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelRecvLength">
           <property name="text">
            <string>数量</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditRecv"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
